flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -644,9 +644,17 @@ extractExecutionParams() {
     local execution_config=$1
     local EXECUTION_PREFIX="BASH_JAVA_UTILS_EXEC_RESULT:"
 
-    if ! [[ $execution_config =~ ^${EXECUTION_PREFIX}.* ]]; then
+    local line_count=`echo "$execution_config" | grep "" -c`
```

Comment:
`taskmanager.sh` uses a different way of counting lines: `num_lines=$(echo "${java_utils_output}" | wc -l)`
I'd prefer if we stick to 1 way of doing it